docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -592,6 +592,15 @@ apply `optimize` only on the partitions corresponding to the filter:
 ALTER TABLE test_partitioned_table EXECUTE optimize
 WHERE partition_key = 1
 ```
+You can use a more complex `WHERE` clause to narrow down the scope of the 
```

Comment:
We now have a shared included for optimize .. I think you probably should move this example into that fragment after rebasing master